src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -1721,6 +1721,8 @@ public void hideBottomComponent() {
     public void highlightEntry(final BibEntry be) {
```

Comment:
Maybe rename` be` to `bibEntry`. I dont see why you should not call it what it is.